extension/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 0.9.0
```

Comment:
This should really go under the "Unreleased" section, as I would bump the version before doing a release.

Since I haven't released anything yet though, this probably doesn't need to go into the changelog.